cmd/command.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// var (
+// 	commandConfig generator.CommandConfig
+// )
+
+// LoadCosignConfigs loads cosign plugin configuration from conf.yaml using "plugins.cosign" key
+func LoadCommandConfigs(cmd *cobra.Command) {
+
+	//g.CommandConfig.CommandConfig = "ls"
+	m, _ := cmd.Flags().GetStringToString("overrides")
+	for k, v := range m {
+		if k == "command" {
+			//fmt.Println("command key found!!!!", v)
+			g.CommandConfig.CommandConfig = v
```

Comment:
where is `g`?

Reply:
Hi @angrieralien Thanks for the review. In command package defined the 'g' variable in generator.go file.
ref: https://github.com/equinixmetal-buildkite/dynamic-buildkite-template/blob/main/cmd/generate.go#L15-L19